base/doc/ltnews33.tex

```diff
@@ -112,6 +112,31 @@ \section{Introduction}
 
 
 
+\section{Extending the hook concept to paragraphs}
+
+Largely triggered by the need for better control of paragraph text
+processing, in particular when producing tagged PDF output (see
+\cite{33:blueprint}), we have extended the paragraph processing of
+\LaTeX{} so that the kernel gains control both at the start and the
+end of each paragraph. This is is done in a manner that is (or should
```

Comment:
typo: `This is is done` -> `This is done`

Reply:
thanks